core/src/main/java/org/apache/calcite/sql/SqlWindowTableFunction.java

```diff
@@ -44,28 +43,21 @@ public SqlWindowTableFunction(String name) {
         SqlFunctionCategory.SYSTEM);
   }
 
-  @Override public SqlOperandCountRange getOperandCountRange() {
-    return SqlOperandCountRanges.of(3);
-  }
-
-  @Override public boolean checkOperandTypes(SqlCallBinding callBinding,
```

Comment:
Why we remove these methods? Each window table function has its own implementation?

Reply:
Yes. Each windowing table-valued function shall check their own operands. The check of DESCRIPTOR names are also left for each function (which makes sense as each function has its own interpretation of names).